go.mod

```diff
@@ -109,4 +109,4 @@ require (
 )
 
 // until merged upstream: https://github.com/omissis/go-jsonschema/pull/264
-replace github.com/atombender/go-jsonschema => github.com/nolag/go-jsonschema v0.16.0-smallSizedInts
+replace github.com/atombender/go-jsonschema => github.com/nolag/go-jsonschema v0.16.2-rtinianov-merge-request
```

Comment:
This is my last official merge before the owner is reviewing.  I got small PR feedback two days ago and addressed it today.  Hopefully I'll remove this soon.

The upgrade allows go-jsonschmea to use remote refs.  It looked like someone accidentally disabled it because it had it before and the structs to do it still existed but weren't used.